assets/base.css

```diff
@@ -1039,11 +1039,20 @@ summary::-webkit-details-marker {
   width: calc(50% - var(--grid-mobile-horizontal-spacing) / 2);
 }
 
-@media screen and (min-width: 750px) {
+@media screen and (max-width: 989px) {
+  .grid--1-col-tablet .grid__item {
+    width: 100%;
+    max-width: 100%;
+  }
+
   .grid--2-col .grid__item {
```

Comment:
@kmeleta I understand what you mean 🤔 . What I am saying is that issue is that `grid--2-col` is already being used in other sections and it currently doesnt have media queries on it.

So I don't think we should touch it on this PR, otherwise we would have to update all of these other sections that arent related to this PR and itd require more testing

Reply:
We can create a follow up to clean up these classes 🤔